src/configuration/utils.ts

```diff
@@ -0,0 +1,18 @@
+import Bugsnag from '@bugsnag/react-native';
+import {useFirestoreConfiguration} from './FirestoreConfigurationContext';
+
+export const useFareProductConfig = (type: string) => {
+  const {fareProductTypeConfigs} = useFirestoreConfiguration();
+
+  const productConfig = fareProductTypeConfigs.find(
+    (config) => config.type === type,
+  );
+
+  if (!productConfig) {
+    const errorMessage = `error trying find a configuration for type: ${type}`;
+    Bugsnag.notify(errorMessage);
+    throw errorMessage;
```

Comment:
@gorandalum this is kind of mandatory, we need these configurations, do you think this is a good way to catch the error?